src/api/db/migrate/20231006084054_create_comment_locks.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCommentLocks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :comment_locks, id: :bigint do |t|
+      t.text :reason, null: false
+      t.string :commentable_type, null: false
+      t.integer :commentable_id, null: false
+
+      t.timestamps
+    end
+
+    add_index :comment_locks, [:commentable_type, :commentable_id], unique: true
+  end
+end
```

Comment:
Could you move the migration to an independent PR? (migration, schema changes, annotations)

Creating a table won't cause any issues in the deployment,  most likely, but it's good to get into the habit of splitting the migration and the related code changes into two PRs, even if that's not the final solution for our problems while deploying migrations.

Reply:
Done, see https://github.com/openSUSE/open-build-service/pull/15041